.github/workflows/unittest.yml

```diff
@@ -0,0 +1,24 @@
+name: run python tests
+on: [push, pull_request]
+
+jobs:
+  python_unittests:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7, 3.8, 3.9]
```

Comment:
I would only test for 2.7, 3.6 and 3.9 as rhel8 uses 3.6 and rhel9 uses 3.9. But it also should not make that much difference.

Reply:
true, I've trimmed it to 2.7, 3.6, 3.9 and 3.11.